### PR TITLE
updating doc versions, images and cleaning up section for prompt-guard

### PR DESCRIPTION
**Pull request by salmanap**
Made several changes to the docs and README based on the hiccups that @peterj faced when going through the docs (vs. the demos on GH).

### CODE REVIEW

File: docs/source/get_started/includes/quickstart.yaml [hunk 1]
```diff
@@ -1,44 +1,43 @@
...
-      - name: device_group
-        type: str
-        description: The name of the device group to reboot
-        required: false
+        description: A list of device identifiers (IDs).
+        required: true
+      - name: days
+        type: int
+        description: A list of device identifiers (IDs)
+        default: "7"
```
Comment (adilhafeez) [inline docs/source/get_started/includes/quickstart.yaml:33]: I think this getting started example isn't great. We can do better. We can have following endpoint,

```
 description: Get device metadata that contains details like sku, up time, num cpus/gpus etc.
    endpoint:
      name: metadata_server
      path: /v1/metadata
    parameters:
      - name: device_ids
        type: list
        description: A list of device identifiers (IDs).
        required: true
```

**Review by adilhafeez — CHANGES_REQUESTED** [review]

File: docs/source/get_started/includes/quickstart.yaml [hunk 2]
```diff
@@ -1,44 +1,43 @@
 version: v0.1
-
-listen:
-  address: 0.0.0.0 # or 127.0.0.1
-  port: 10000
-  # Defines how Arch should parse the content from application/json or text/pain Content-type in the http request
+listener:
+  address: 127.0.0.1
+  port: 8080 #If you configure port 443, you'll need to update the listener with tls_certificates
```
[the remark this answers is not in the document]
Comment (salmanap) [reply docs/source/get_started/includes/quickstart.yaml:4]: Hmmm. Is that new? All demos use listener. And not listeners

[the remark this answers is not in the document]
Comment (adilhafeez) [reply docs/source/get_started/includes/quickstart.yaml:4]: Ah wait it's listener. Not sure what I was thinking 😵‍💫

File: docs/source/get_started/includes/quickstart.yaml (hunk 1 above)
Comment (salmanap) [reply docs/source/get_started/includes/quickstart.yaml:33]: Fair. In this instance I am trying to match our README with docs. I'll open an another issue to improve getting started

**Pull request merged**